projects/JLabMOLLER.yaml

```
@@ -0,0 +1,10 @@
+Department: Physics
+Description: Jefferson Lab's MOLLER experiment
+FieldOfScience: Nuclear Physics
+ID: '808'
+Name: MOLLER
```

```suggestion
Name: JLabMOLLER
```

Name and the filename without the `.yaml` suffix should match